articles/140_topic_and_service_name_mapping.md

```diff
@@ -219,13 +219,83 @@ Since ROS 2 topic and service names are expanded to fully qualified names, any b
 Additionally any URL related syntax, e.g. the `rostopic://` prefix, will be removed once parsed.
 Therefore only forward slashes have to be substituted when converting to DDS topic names.
 
-### Substitution of the Namespace Delimiter
+### DDS Partitions as namespace hierarchy
 
-The namespace delimiter in ROS 2 topic and service names, a forward slash (`/`), will be replaced with double underscores (`__`).
-Note that as fully qualified ROS 2 topic and service names are absolute, there is always a leading forward slash (`/`).
+Having looked closer to the various QoS settings inside the DDS protocol, paritions have been considered as the prefered method of implementing namespaces.
+DDS partitions allow up to 64 items in a string array, limiting the accumulated string size to 255 characters (TODO: Find spec ref).
+The big benefit of handling the namespaces with partitions is that it generally stays ROS agnostic.
+This means if future implementations of the RMW interface (e.g. ZeroMQ etc.) want to introduce another, different namespace system, they are non-restricted.
+
+#### DDS Partitions Characteristics
+
+As already said, DDS Partitions are implemented as a QoS setting.
+Hereby, the partitions field is implemented as an array of strings, where each string is an individual partitions.
+This means further that every partition index is stricly unique and independent from all other items.
+At the same time, DDS Partitions don't support a partition hierarchy.
+If a publisher operates on two partition entries, e.g. `foo` and `bar` with a base name of `baz`, the resulting topic will be twofold:
+
+```
+- /foo/baz
+- /bar/baz
+```
+
+That implies, that DDS Partitions by default are not depicting any hierarchy.
+However, aliasing is easily douable with it.
+That is, having an original topic such as `cmd_vel` in a namespace `robot1`, we can easily duplicate that topic as `robot2/cmd_vel`.
```

Comment:
How does this easy way of aliasing extend to aliasing including the basename? Are we expecting to expose this sort of aliasing to the user through the API?